cfme/tests/test_db_migrate.py

```diff
@@ -29,34 +36,39 @@ def pytest_generate_tests(metafunc):
         version.get_stream(db_version) == version.current_stream())
 @pytest.mark.meta(
     blockers=[BZ(1354466, unblock=lambda db_url: 'ldap' not in db_url)])
-def test_db_migrate(db_url, db_version, db_desc, v2key_url):
+def test_db_migrate(stabilize_current_appliance, db_url, db_version, db_desc):
     """ This is a destructive test - it _will_ destroy your database """
```

Comment:
My only concern here is that some "session" based fixtures would be effectively nullified by this blowing away the db, would it be better for us to provision a module based appliance for this testing as part of a fixture?